packages/@aws-cdk/aws-glue/lib/connection.ts

```diff
@@ -0,0 +1,114 @@
+import ec2 = require('@aws-cdk/aws-ec2');
+import { Aws, Construct, IResource, Resource } from '@aws-cdk/core';
+import { CfnConnection } from './glue.generated';
+
+/**
+ * Types of glue input connections
+ */
+export enum ConnectionInputTypes {
+  /**
+   * The type of the connection. Currently, only JDBC is supported;
+   * SFTP is not supported.
+   */
+  JDBC = 'JDBC',
+}
+
+export interface IConnection extends IResource {
+  /**
+   * The name of the connection
+   */
+  readonly connectionName: string;
+}
+
+export interface ConnectionAttributes {
```

Comment:
This interface is never used - feel free to remove for now (I assume you'e gearing up towards a `fromConnectionAttributes` method with it).

Reply:
removed for now